datumaro/plugins/open_images_format.py

```diff
@@ -591,6 +723,58 @@ def _save_subsets(self, annotation_writer):
                                     for bool_attr in OpenImagesPath.BBOX_BOOLEAN_ATTRIBUTES
                                 },
                             })
+                        elif annotation.type is AnnotationType.mask:
+                            mask_dir = osp.join(self._save_dir, OpenImagesPath.MASKS_DIR, item.subset)
+
+                            box_id_str = annotation.attributes.get('box_id')
+
+                            if box_id_str:
+                                if _RE_INVALID_PATH_COMPONENT.fullmatch(box_id_str):
+                                    raise UnsupportedBoxIdError(item_id=item.id, box_id=box_id_str)
+                            else:
+                                # find a box ID that isn't used in any other annotations
+                                while True:
+                                    box_id_str = format(next_box_id, "08x")
+                                    next_box_id += 1
+                                    if box_id_str not in existing_box_ids:
+                                        break
+
+                            label_name = label_categories[annotation.label].name
+                            mask_file_name = '%s_%s_%s.png' % (
+                                item.id, make_file_name(label_name), box_id_str,
+                            )
+
+                            box = annotation.attributes.get('box')
+                            box_coords = {}
+
+                            if box is not None:
+                                if item.has_image and item.image.size is not None:
+                                    image_meta[item.id] = (height, width) = item.image.size
```

Comment:
I discourage using multiple assignment pattern.

Reply:
Why?

Reply:
Basically, for the same reason we don't write lines like `do_one(); do_another(); x = foo(); y = bar();`. This make the code clearer and reduces load on the reader.

Reply:
It's not quite the same thing, because it reduces duplication. But eh, I can change it if you want.

Reply:
Removed.